SetupDataPkg/Tools/ConfigEditor.py

```diff
@@ -1092,8 +1291,8 @@ def add_config_item(self, item, row):
 
         if widget:
             create_tool_tip(widget, item["help"])
-            self.set_object_name(name, "LABEL_" + item["path"])
-            self.set_object_name(widget, item["path"])
+            self.set_object_name(name, "LABEL_" + item["path"], file_id)
+            self.set_object_name(widget, item["path"], file_id)
```

Comment:
btw, did you verify to see if the "help" texts still look decent after the change? That is when you hover the mouse over available options.

Reply:
Yes, I confirmed the help text shows up as expected.